src/d3a/global_utils.py

```diff
@@ -0,0 +1,9 @@
+from d3a.constants import BidOfferMatchAlgoEnum
+from d3a_interface.constants_limits import ConstSettings
+
+
+def check_is_exact_match_type(bid_offer_match_algo: BidOfferMatchAlgoEnum):
+    """Compares the matching algorithm set in IAASettings with the passed bid_offer_match_algo
+    Returns True if both are matched
+    """
+    return ConstSettings.IAASettings.BID_OFFER_MATCH_TYPE == bid_offer_match_algo.value
```

Comment:
IMO this way of implementing it does not make the check shorter or more readable. You actually only want to check if the match type is custom right? There will not be any other custom match algorithms in the future, so I propose:
```suggestion
def is_custom_matching_enabled():
    return ConstSettings.IAASettings.BID_OFFER_MATCH_TYPE == d3a.constants.BidOfferMatchAlgoEnum.CUSTOM and self.simulation_config.external_connection_enabled
```
I also added the check for the `external_connection_enabled` to make the call in the simulation.py even shorter

Reply:
What happened to my comment `I also added the check for the external_connection_enabled to make the call in the simulation.py even shorter` ? Didnt you like that?

Reply:
@hannesdiedrich the `external_connection_enabled` attribute exists in the `self.simulation_config` object which does not exists in the function, we can pass it to it but this won't make the code shorter at all , it's basically the same

Reply:
(For future PR reviews: This comment would have bee great even before I asked for it. Thanks)

I understand. Then leave it out. Sorry for dragging this cosmetic issue so long. I thought is is a simple change and didnt thing it is such a hassle.